### ISSUE TITLE
👉 Official Intercom React Native Library 👉

### ISSUE THREAD

**Issue by nabilfreeman** [tinycreative/react-native-intercom#459]
Big news! Intercom has an official React Native library:

https://github.com/intercom/intercom-react-native

I didn't know this existed and was thinking about making some updates to this library here to be compatible with some of Intercom's newer features.

Now I'll try the official one out before opening any PRs here.

If anybody finds some niche functionality missing from the official one, please leave a comment on this issue.

**Comment by datoon83** [comment]
I've just tried to migrate to the official package.  However, I can't for the life of me get Push Notifications working on iOS, I managed to eventually get them working on Android - there is an open PR and missing documentation.  

If you try to do a push notification on the simulator on iOS it just won't work.  I've now reverted to a previous version using this library (not the official one) and it seems to be working now.

Not sure whether other people have migrated or tried iOS Push Notifications and got them working.
